### ISSUE TITLE
QSlider triggers @mouseleave when dragging

### ISSUE THREAD

**Issue by Blaconix** [quasarframework/quasar#7739]
**Describe the bug**
I'm trying to wrap a QSlider inside a QChip so the slider is only visible when you hover over the QChip. The practical use is in chrome's default video player - when you hover over the volume icon, the slider appears with a transition.

The problem that I'm encountering is when you drag the thumb, @mouseleave gets triggered (don't know how it happens). It seems to be a Quasar issue (tried it with vuetify with the same code and the behavior doesn't happen).

EDIT: On phone doesn't happen.

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/Blaconix/pen/ExKoMmj

**Expected behavior**
@mouseleave shouldn't be triggered and the QChip should remain expanded.

**Comment by rstoenescu** [comment]
QSlider does not emit any mouseleave event. It's because of a different reason: QSlider uses our TouchPan under the covers, which adds a pointer-events: none to all children of the body (so that panning will not be interfered by anything else on the page; and to also be able to drag the mouse cursor anywhere on the page while still maintaining the grabbing state; and also to be able to display the grabbing cursor safely; plus other reasons in regards to different platforms --- a full explanation is equivalent to a book in itself).

So, added `@pan` event to QSlider (also to QRange) which is triggered with a String param which can either be 'start' or 'end'. In your case, you can take advantage of this:

```
<q-chip class="q-pa-none" @mouseover.native="show = true" @mouseleave.native="show = false">
  <q-btn :icon="volume ? 'volume_up' : 'volume_mute'" flat round name="volume_up" @click='volume ? volume = 0 : volume = 20'></q-btn>
  <q-slider v-if="show || panning" @pan="onPanning" class="q-mx-sm" v-model="volume" :min="0" :max="100" style="width: 100px"></q-slider>
</q-chip>

export default {
  data () {
    return {
      volume: 20,
      show: false,
      panning: false
    }
  },
  methods: {
    onPanning (phase) {
      this.panning = phase === 'start'
        ? true
        : false
    }
  }
}
```

This will be available in Quasar v1.13.3.

**Comment by Blaconix** [comment]
Great fix, thanks a lot!
